app/app.go

```diff
@@ -133,7 +133,7 @@ func Run() error {
 			{
 				config, err := evm.NewEVMConfig(chainConfig)
 				panicOnError(err)
-
+				log.Info().Msgf("Loaded config %s", config.String())
```

Comment:
This is already implemented here https://github.com/sygmaprotocol/sygma-relayer/blob/f246f89e2a871e8e0e8c2a3c7d9c7dd6b4f43e97/app/app.go#L143